### ISSUE TITLE
Event rendering problem

### ISSUE THREAD

**Issue by Sehur** [angular-ui/ui-calendar#345]
Im having problems with the rendering of the events at the calendar, Im gettint the events from a database and sometimes they get displayed at the calendar but sometimes this error shows up:

TypeError: Cannot read property 'length' of null

Im not sure why sometimes it works and sometimes it doesnt.

**Comment by Sehur** [comment]
In case someone gets my same mistake, I had an error in the changeLang() method, I've just noticed that when I changed the language the calendar started working.

My mistake

**Comment by mariohmol** [comment]
is this the case to close the issue?

**Comment by Sehur** [comment]
Yeah it was the render problem that was solved in another thread, thanks
